### DISCUSSION TITLE
WiinUPro and Windows won't detect my WiiU Pro Controller

### DISCUSSION THREAD

**Discussion by cedriclenavet** [KeyPuncher/WiinUPro#154] [Help] [answered]
Hi, i got a problem
I installed WiinUPro and tried to connect my wii U pro controller to WiinUPro.
But... It didn't detect the controller. So I tried to pair the controller with Windows Bluetooth settings, and even this doesn't recognize the controller, it don't even shows a bluetooth device.
My bluetooth is not broken : i paired another bluetooth device and it worked...
Does anybody knows where the problem comes from ?

**Comment by KeyPuncher** [comment]
Sounds like the behavior of a third party Wii U Pro controller.

**Comment by cedriclenavet** [comment]
Oops... Is there a way to know is my controller is a third-party one ?

**Comment by DraakjeYoblama** [comment]
You could probably check the text on the back to see if it's original

**Comment by KeyPuncher** [comment]
From what I've seen, the 3rd party clones don't have the Wii U logo on the front above the home button.

**Comment by Willicious** [comment, marked as answer]
I'm having the same issue, although my controller did actually connect the first time I used the program (and is definitely a legit Nintendo controller). Thereafter, I got a "Failed to pair" error in WiinUSoft so I removed the controller from Windows Bluetooth Manager, and now I can't pair it at all from either WiinUSoft or WIndows itself.

EDIT: OK, I got it working using [this solution](https://answers.microsoft.com/en-us/windows/forum/all/cant-connect-wii-motion-plus-remote-on-windows-10/c430affc-39cd-4f17-a1e4-3fd8d613f500). Reinstalling WiinUSoft and choosing "Install Everything" also seemed to help.
